tests/evm_tools/test_b11r.py

```diff
@@ -0,0 +1,88 @@
+import json
+import os
+from typing import Any, Dict, List
+
+import pytest
+
+from ethereum.base_types import U64, U256, Uint
+from ethereum.rlp import decode
+from ethereum.utils.hexadecimal import (
+    Hash32,
+    hex_to_bytes,
+    hex_to_u256,
+    hex_to_uint,
+)
+from ethereum_spec_tools.evm_tools import parser, subparsers
+from ethereum_spec_tools.evm_tools.b11r import B11R, b11r_arguments
+from ethereum_spec_tools.evm_tools.utils import FatalException
+from tests.helpers import TEST_FIXTURES
+
+B11R_TEST_PATH = TEST_FIXTURES["evm_tools_testdata"]["fixture_path"]
+
+ignore_tests: List[str] = []
+
+
+def find_test_fixtures() -> Any:
+    with open(os.path.join(B11R_TEST_PATH, "b11r_commands.json")) as f:
+        data = json.load(f)
+
+    for key, value in data.items():
+
+        final_args = []
+        for arg in value["args"]:
+            if "__BASEDIR__" in arg:
+                final_args.append(arg.replace("__BASEDIR__", B11R_TEST_PATH))
+            else:
+                final_args.append(arg)
+        # if key == "b11r/fixtures/expected/22/exp.json":
```

Comment:
Is this intentionally commented out?

Reply:
Ah. This line was used for dev. Might have missed deleting it.